Pogo Data Generator

TurtIeSocks opened the pull request:
- Updated to use the new pogo data generator module
- Now includes two versions, fully parsed and raw
- Raw is for pogo data generator to grab a known safe masterfile when running in safe mode

TurtIeSocks committed:
Pogo Data Generator

- Updated to use the new pogo data generator module
- Now includes two versions, fully parsed and raw
- Raw is for pogo data generator to grab a known safe masterfile when running in safe mode

Mygod commented on generate.js:
```diff
@@ -1,548 +1,236 @@
-                Form.stamina = object.data.pokemonSettings.stats.baseStamina;
-                if (['_NORMAL', '_SHADOW', '_PURIFIED'].some(suffix => object.templateId.endsWith(suffix))) {
-                  console.warn('eMbArRaSsInG', object.templateId);
-                }
```
y u remove my check :angryeyes:

TurtIeSocks committed:
Less Hacky Top Level Names

TurtIeSocks committed:
Pogo Data Generator v1.1.0 Update

- Fixes Nidoran
- Normal/Purified Estimated Megas
- Few misnamed fields
- False values are now included
- Form unsets are skipped

TurtIeSocks committed:
Consistency

- More consistent with old generator
- Update protos
- Includes unset for master-latest-v2

TurtIeSocks committed:
Update generate.js

TurtIeSocks committed:
oops

TurtIeSocks committed:
Item Adjustments

- Raise min level
- Fix type names

TurtIeSocks committed:
Host All Masterfiles

- Change of plans, now going to use this repo to host all masterfiles for JS projects, using an assortment of templates
- Support for usual latest version using the standard format, basics for ohbem, poracle, reactmap, rdmopole2 and a raw version

TurtIeSocks committed:
add-invasions-to-react-map

TurtIeSocks committed:
buddy distance

Mygod reviewed (commented):
I have lost hope in checking everything in this PR after so many back and forths. I recommend merge and blame @TurtIeSocks if anything goes wrong.
